daml-assistant/daml-helper/BUILD.bazel

```diff
@@ -106,6 +106,7 @@ $(location @openssl_dev_env//:openssl) x509 -req -in $(location client.csr) -CA
     tools = [
         "@openssl_dev_env//:openssl",
     ],
+    visibility = ["//visibility:public"],
```

Comment:
If they're used outside `daml-assistant`, maybe they should live in a more global location?

Reply:
Yeah, they are currently  a bunch of test certificates all over the place. I’m planning to remove all of them and move the one from the assistant (which is the only one that works for openssl and is generated by a genrule rather than checked in) to a more central location.